docs/custom.css

```diff
@@ -8,7 +8,102 @@
   transition: unset;
   box-shadow: 0 0 0.2rem rgba(0, 0, 0, 0.1), 0 0.2rem 0.4rem rgba(0, 0, 0, 0.2);
 }
-
-[data-md-color-primary=grey] .md-header, [data-md-color-primary=grey] .md-hero {
+button[data-md-color-primary="larq"] {
+  background-color: #0a1d3c;
+}
+[data-md-color-primary="larq"] .md-typeset a {
+  color: #153c7a;
+}
+[data-md-color-primary="larq"] .md-header,
+[data-md-color-primary="larq"] .md-hero {
+  background-color: #0a1d3c;
+}
+[data-md-color-primary="larq"] .md-nav__link--active,
+[data-md-color-primary="larq"] .md-nav__link:active {
+  color: #153c7a;
+}
+[data-md-color-primary="larq"] .md-nav__item--nested > .md-nav__link {
+  color: inherit;
+}
+button[data-md-color-accent="larq"] {
+  background-color: #7dfb9a;
+}
+[data-md-color-accent="larq"] .md-typeset a:active,
+[data-md-color-accent="larq"] .md-typeset a:hover {
+  color: #059e29;
+}
+[data-md-color-accent="larq"]
+  .md-typeset
+  .codehilite
+  pre::-webkit-scrollbar-thumb:hover,
+[data-md-color-accent="larq"]
+  .md-typeset
+  pre
+  code::-webkit-scrollbar-thumb:hover {
+  background-color: #059e29;
+}
+[data-md-color-accent="larq"] .md-nav__link:focus,
+[data-md-color-accent="larq"] .md-nav__link:hover,
+[data-md-color-accent="larq"]
+  .md-typeset
+  .footnote
+  li:hover
+  .footnote-backref:hover,
+[data-md-color-accent="larq"] .md-typeset .footnote li:target .footnote-backref,
+[data-md-color-accent="larq"] .md-typeset .md-clipboard:active:before,
+[data-md-color-accent="larq"] .md-typeset .md-clipboard:hover:before,
+[data-md-color-accent="larq"] .md-typeset [id] .headerlink:focus,
+[data-md-color-accent="larq"] .md-typeset [id]:hover .headerlink:hover,
+[data-md-color-accent="larq"] .md-typeset [id]:target .headerlink {
+  color: #059e29;
+}
+[data-md-color-accent="larq"]
+  .md-search__scrollwrap::-webkit-scrollbar-thumb:hover {
+  background-color: #059e29;
+}
+[data-md-color-accent="larq"] .md-search-result__link:hover,
+[data-md-color-accent="larq"] .md-search-result__link[data-md-state="active"] {
   background-color: #0a1d3c;
 }
+[data-md-color-accent="larq"]
+  .md-sidebar__scrollwrap::-webkit-scrollbar-thumb:hover {
+  background-color: #059e29;
+}
+[data-md-color-accent="larq"] .md-source-file:hover:before {
+  background-color: #059e29;
+}
+@media only screen and (max-width: 59.9375em) {
```

Comment:
It seems weird to me to have `em`/`rem` dimensions which are so specific. Not an issue, just interesting! It looks like it's been calculated as 959px / 16pt font size . I guess doing it with `em` improves accessibility for non-default font-sizes.

Reply:
I am also unsure about this. This is mostly copy paste from the default theme, so I don't know what the exact reasoning is behind this.